meso-org/meso-ui
# License View + logo

## Description 
Login section that asks user to input their licenses and put in logo where needed

## Testing 

* localhost:8080, look at the logo!
* go thru the signup process
* When u get to the license section: 
* put in a license
* Put another
* click on the x to get rid of field
* add as many as u can (u wont be able to do more than 4 :)
* Click verify all
* Click the red exes to get rid of em
* check that multiple viewpoints work
*finish the signup process
